### PR TITLE
Dont start Riak in TP/TO GHA

### LINKED ISSUE #5946: TO API Tests GHA still starts Riak container
## I'm submitting a ...

-  improvement request (usability, performance, tech debt, etc.)

## Traffic Control components affected ...

- GitHub Actions

## Current behavior:

The *to-integration-tests* GitHub Action starts but does not use a Riak Docker container.

## New behavior:

The Riak container should be removed.

## Minimal reproduction of the problem with instructions:

https://github.com/apache/trafficcontrol/blob/295c31e3467d13163ec4a3f87fac5450389801ec/.github/actions/to-integration-tests/entrypoint.sh#L87-L92

### PR DESCRIPTION
**Pull request by shamrickus**
This PR fixes #5946 and #5947
<hr>

## Which Traffic Control components are affected by this PR?

- Automation (GHA) 

## What is the best way to verify this PR?
Inspect the logs and make sure traffic vault is not running. For TP, look at the artifacts and ensure that `tv.log` does not exist.

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: .github/actions/tvdb-init/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-->
+
+# tvdb-init Docker action
+This action initializes a Traffic Vault database in a locally running PostgresQL
+instance. It creates the `traffic_vault` user and database.
+
+## Outputs
+
+### `exit-code`
+1 if the database initialized successfully.$^
```
Comment (zrhoffman) [inline .github/actions/tvdb-init/README.md:27]: `$^`

**Review by zrhoffman — CHANGES_REQUESTED** [review]

**Review by zrhoffman — CHANGES_REQUESTED** [review]
No longer provides a Riak server
https://github.com/apache/trafficcontrol/blob/67904798c8f01dce7d913e1f23b078fa32f7172a/.github/actions/to-integration-tests/README.md#L23

There is no longer a Riak server
https://github.com/apache/trafficcontrol/blob/67904798c8f01dce7d913e1f23b078fa32f7172a/.github/actions/tp-integration-tests/README.md#L23

Is inserting a `RIAK` server still necessary?
https://github.com/apache/trafficcontrol/blob/67904798c8f01dce7d913e1f23b078fa32f7172a/.github/actions/tp-integration-tests/entrypoint.sh#L90-L103

No need to gitignore `riak.conf`
https://github.com/apache/trafficcontrol/blob/67904798c8f01dce7d913e1f23b078fa32f7172a/.github/actions/to-integration-tests/.gitignore#L24

File: .github/actions/to-integration-tests/cdn.json [hunk 2]
```diff
@@ -22,6 +22,20 @@
...
+			"hostname": "localhost",
+			"user": "traffic_vault",
+			"password": "twelve",
+			"port": 5432,
+			"ssl": false,
+			"conn_max_lifetime_seconds": 60,
+			"max_connections": 500,
+			"max_idle_connections": 30,
+			"query_timeout_seconds": 10,
+			"aes_key_location": "/aes.key"
```
Comment (ocket8888) [inline .github/actions/to-integration-tests/cdn.json:37]: Don't you only need this for retrieving things from a Hashicorp vault? Do the integration tests need to do that?

File: .github/workflows/tp.integration.tests.yml [hunk 3]
```diff
@@ -107,6 +109,9 @@ jobs:
       - name: Initialize Traffic Ops Database
         id: todb
         uses: ./.github/actions/todb-init
+      - name: Initialize Traffic Vault Database
+        id: tvdb
+        uses: ./.github/actions/tvdb-init
```
Comment (ocket8888) [inline .github/workflows/tp.integration.tests.yml:114]: Could we use the same PostgreSQL instance as the TODB but just add another database? Seems like that might be faster.

File: .github/actions/to-integration-tests/cdn.json (hunk 2 above)
Comment (ocket8888) [reply .github/actions/to-integration-tests/cdn.json:37]: No, I'm wrong about that.

File: .github/workflows/tp.integration.tests.yml (hunk 3 above)
Comment (shamrickus) [reply .github/workflows/tp.integration.tests.yml:114]: That is what this does, it logs in using `traffic_ops` and creates `traffic_vault` user and db

Comment (ocket8888) [reply .github/workflows/tp.integration.tests.yml:114]: Yeah, I see that now. My bad.

**Review by ocket8888 — APPROVED** [review]
Somehow didn't see the existing reviewer. The two things I pointed out weren't real, so I'm bowing out now.

**Review by zrhoffman — APPROVED** [review]
Looks good! Verified that Traffic Vault is not running, and, for TP, no `traffic_vault.log` file exists in the artifact on failure.

**Pull request merged**